src/backend/distributed/commands/multi_copy.c

```diff
@@ -1051,6 +1051,9 @@ RemoteFinalizedShardPlacementList(uint64 shardId)
 			shardPlacement->nodeName = nodeName;
 			shardPlacement->nodePort = nodePort;
 
+			/* we cannot know the nodeId, but it is not necessary at this point as well */
```

Comment:
as well -> either

or -> because it is only used to to look up the connection for a group of co-located placements, but append-distributed tables are never co-located.